src/Microsoft.Azure.WebJobs.Host/Executors/StorageAccountProviderExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static Task<IStorageAccount> GetDashboardAccountAsync(this IStorageAccoun
                 throw new ArgumentNullException("provider");
             }
 
-            return provider.GetAccountAsync(ConnectionStringNames.Dashboard, cancellationToken);
+            return provider.GetAccountAsync(ConnectionStringNames.Dashboard, StorageAccountTypes.GeneralPurpose, cancellationToken);
```

Comment:
I think you should have a default parameter for the account type, so for the mainline case you don't have to modify the code everywhere. We should only have to pass in this parameter in cases where you need to restrict.